AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/list/participant-list.component.ts

```diff
@@ -109,4 +104,20 @@ export class ParticipantListComponent implements OnInit, OnChanges {
             i.is_interpretee = false;
         });
     }
+
+    canEditParticipant(particpant: ParticipantModel): boolean {
```

Comment:
Can this be simplified?

Reply:
This is probably as simplified as it can be

Reply:
```
        if (!this.canEdit || this.videoHearingsService.isConferenceClosed()) {
            return false;
        }

        if (!this.videoHearingsService.isHearingAboutToStart() && !particpant.addedDuringHearing) {
            return false;
        }

        return true;
```

Reply:
or

```
        if (!this.canEdit || this.videoHearingsService.isConferenceClosed()) {
            return false;
        }

        return this.videoHearingsService.isHearingAboutToStart() || particpant.addedDuringHearing;
```

Reply:
Yeah I like the first option, implemented

Reply:
Does the second not work?